tests/integration/test_github.py

```diff
@@ -359,6 +359,9 @@ def test_get_web_url(self):
     def test_full_repo_name(self):
         assert self.ogr_project.full_repo_name == "packit-service/ogr"
 
+    def test_is_private(self):
+        assert not self.ogr_project.is_private()
```

Comment:
Is it possible to test some private repository? Is it possible to create one in our organisation?

Reply:
Our current plan does not support private repositories. We can create it on one of our users, but while re-generating the test data, the person would have to provide their own repository with their github key.

Reply:
OK, but we don't need to share the keys. Just create the repository with the same name and use that repository in tests. (The namespace should be the user of the token.)